### PR TITLE
Service

### PR DESCRIPTION
**Pull request by adrissonGalvao**

### CODE REVIEW

File: service/sensor.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	var sensor domain.Sensor
+
+	if err := json.NewDecoder(r.Body).Decode(&sensor); err != nil {
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	sensor.ID = bson.NewObjectId()
+	if err := sensorRepository.InsertSensor(sensor); err != nil {
+		respondWithError(w, http.StatusBadRequest, "Invalid request")
+	}
```
Comment (edvmorango) [inline service/sensor.go]: @adrissonGalvao the missing "return" is intentional?

File: service/sensor.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	defer r.Body.Close()
+
+	var sensor domain.Sensor
+
+	if err := json.NewDecoder(r.Body).Decode(&sensor); err != nil {
+		respondWithError(w, http.StatusBadGateway, err.Error())
+		return
+	}
+	if err := sensorRepository.UpdateSensor(sensor); err != nil {
+		respondWithError(w, http.StatusInternalServerError, err.Error())
```
Comment (edvmorango) [inline service/sensor.go:75]: @adrissonGalvao the missing "return" is intentional?

File: service/sensor.go (hunk 1 above)
Comment (adrissonGalvao) [reply service/sensor.go]: I fixed it.

File: service/sensor.go (hunk 2 above)
Comment (adrissonGalvao) [reply service/sensor.go:75]: I fixed it

**Pull request merged**